geode-core/src/integrationTest/java/org/apache/geode/cache/PoolManagerIntegrationTest.java

```diff
@@ -103,34 +109,62 @@ public void testRegionFind() {
     ((PoolFactoryImpl) cpf).setStartDisabled(true);
     Pool pool = cpf.addLocator("localhost", 12345).create("mypool");
     Cache cache = CacheFactory.create(ds);
-    AttributesFactory fact = new AttributesFactory();
+    AttributesFactory<Object, Object> fact = new AttributesFactory<>();
```

Comment:
Is there any value to adding Object type parameters here?  I'm in favor of generics where appropriate, but in this case it isn't adding any type enforcement since the two type parameters can be anything that derives from Object.  Could we use more specific types here?

Reply:
We are not using the parameters at all so in the test so no, no value other than removing those **annoying** _unchecked assignment_ warnings... what do you think would be the best here to get rid of these warnings in the tests?, use `?` instead of `Object` in the typed parameters?, or simply add `@SuppressWarnings("unchecked")` to every method that requires it because **we know we don't need the actual parameter types**?.

Reply:
Ok that makes sense.  I think this is a step in the right direction.  These tests are actually using pretty old API and I think could use a full revamp, but that is out of scope for your current story I think.